### ISSUE TITLE
showing parameters window using bind keys isn't working 

### ISSUE THREAD

**Issue by areda0** [RTimothyEdwards/magic#324]
![Screenshot from 2024-09-03 15-07-07](https://github.com/user-attachments/assets/ed7db2e9-c74d-42bc-811c-ab92d88f68ae)

cannot get the parameters window of a selected device to show-up using the bind key "ctrl+p" . attached is the error message on magic console. 

magic version: 8.3.489

**Comment by d-m-bailey** [comment]
@areda0 What command and options did you use to start magic?

**Comment by areda0** [comment]
@d-m-bailey  
magic -d XR -rcfile $PDK_ROOT/$PDK/libs.tech/magic/sky130A.magicrc

**Comment by d-m-bailey** [comment]
@areda0 That should work as long as PDK is `sky130A`. If you use
```
magic -d XR -rcfile $PDK_ROOT/$PDK/libs.tech/magic/$PDK.magicrc
```
it should work with `sky130B` also.

**Comment by areda0** [comment]
the PDK is correctly defined.. using sky130A and still cannot get the bind key to work as expected. 
side note: I installed magic from source

**Comment by RTimothyEdwards** [comment]
I seem to have managed to mess up the "cellname property <cell_name> <key>" command, which is no longer returning a value for the property.

**Comment by RTimothyEdwards** [comment]
Presumably this one.  I made corrections to the "property" command syntax parsing, but did not consider that the same code is called from the "cellname property" command, with a different index into the command arguments.  I will get this fixed ASAP.  Sorry for the delay.

```
commit b441d668bd11e261a61c999c6c5ab55ffcc0da5f
Author: Tim Edwards <tim@opencircuitdesign.com>
Date:   Sat Aug 3 12:05:41 2024 -0400

    Corrected the "property" command parsing, which had a print statement
    without an argument, resulting in random things getting printed to
    the output when a property name does not exist.
```

**Comment by RTimothyEdwards** [comment]
Fixed in magic version 8.3.492 (see commit notes).
@areda0 :  Thank you for logging the issue.
